orca-core/src/main/groovy/com/netflix/spinnaker/orca/TaskResult.java

```diff
@@ -14,20 +14,14 @@
 
   ExecutionStatus status;
   ImmutableMap<String, ?> stageOutputs;
-  ImmutableMap<String, ?> globalOutputs;
```

Comment:
Removed the `globalOutputs` property from `TaskResult`. Any tasks that were using this are now putting the same values in the stage context instead (most of them were duplicating anyway).